### PR TITLE
Update fish recipe for fish 2.2.0 release

### PR DESCRIPTION
**Pull request by ridiculousfish**
fish 2.2.0, get it while it's hot

### CODE REVIEW

File: Library/Formula/fish.rb [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-  sha1 "f7f8d8d26721833be3458b8113c74b747296ec0b"
-
-  bottle do
-    revision 1
-    sha1 "7560818f385831e3d18be1458b1c5e52216f121c" => :yosemite
-    sha1 "a158ae57a437e5f8b2fdff88177be13fa6f35502" => :mavericks
-    sha1 "8d0aa59ebb4cf446e0f0fdf0f91738ffee7edbae" => :mountain_lion
-  end
+  url "http://fishshell.com/files/2.2.0/fish-2.2.0.tar.gz"
+  sha1 "cd7935b3301444afa88afb272d8db5542085c9f2"
```
Comment (bfontaine) [inline Library/Formula/fish.rb]: Would you mind using a `sha256` here instead?

Comment (zanchey) [inline Library/Formula/fish.rb]: sha256 is a76339fd14ce2ec229283c53e805faac48c3e99d9e3ede9d82c0554acfc7b77a

**Comment by geoff-codes** [comment]
Damnation. https://github.com/fish-shell/fish-shell/pull/2183.

**Comment by zanchey** [comment]
Need to nuke the `pkill fishd || true` bit if you're taking this PR.

**Comment by DomT4** [comment]
Let's centralise this in #41631, which seems to be where discussion on the bump is happening.

**Pull request closed without merging**